docs/docker-logs.md

```diff
@@ -0,0 +1,36 @@
+# Docker troubleshooting
```

Comment:
you have two docks with the same name. This one and docs/docker-troubleshooting.md

Reply:
And the docs are the same

Reply:
removed